tests/assign/toverload_asgn2.nim

```diff
@@ -1,7 +1,7 @@
 discard """
   output: '''i value 88
 2aa'''
-  disabled: "true"
+    disabled: "true"
```

Comment:
Disabled just for CI's sake around 2018-04. 
		at the timeline, very near fixes to issues #7637, #6393, altough doesn't seem related to any of them.
		This test doesn't work, but seems fixable. 
		Running it gives the log:
		Expected:
			i value 88
			2aa

			Gotten:
			i value 88
			0aa

Reply:
No, I don't think it is related, or at least I don't see it immediately - how `=` is handled is another can of worms that I should probably check as well.